src/libsyntax/codemap.rs

```diff
@@ -1314,6 +1314,31 @@ impl CodeMap {
         return a;
     }
 
+    /// Check if the backtrace `subtrace` contains `suptrace` as a prefix.
+    pub fn more_specific_trace(&self,
```

Comment:
NIT: the name seems to imply that this method ought to produce a more specific backtrace. Perhaps “is_supertrace” or something would be more suitable?